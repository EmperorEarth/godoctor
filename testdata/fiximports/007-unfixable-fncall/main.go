package main // <<<<< fiximports,1,1,1,1,fail

import (
	"fmt"
	"math/rand"
)

func main() {
  r := rand.New(rand.NewSource(99))
  fmt.Println(r.Float32())
  bytes()
}
