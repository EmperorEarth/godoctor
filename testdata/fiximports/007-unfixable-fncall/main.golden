package main // <<<<< fiximports,1,1,1,1,pass

import (
	"fmt"
	"math/rand"
)

func main() {
  r := rand.New(rand.NewSource(99))
  fmt.Println(r.Float32())
  bytes()
}
